ui/apps/platform/src/Containers/AccessControl/AuthProviders/authProviders.utils.ts

```diff
@@ -212,5 +212,5 @@ export function getDefaultRoleByAuthProviderId(groups: Group[], id: string): str
     if (defaultRoleGroups.length) {
         return defaultRoleGroups[0].roleName;
     }
-    return 'Admin';
+    return id ? 'Undefined, assuming None' : 'Admin';
```

Comment:
What is the use case for the `: 'Admin'` case?

Reply:
It is selected when creating a new provider from the UI.